quinn/src/connection.rs

```diff
@@ -118,8 +118,10 @@ impl Future for ConnectionDriver {
 
 /// A QUIC connection.
 ///
-/// If a `Connection` is dropped without being explicitly closed, it will be automatically closed
-/// with an `error_code` of 0 and an empty `reason`.
+/// If a `Connection` and other references to the underlying state (including `IncomingStreams`
+/// and the various stream types) other than the `ConnectionDriver` have been dropped, the
+/// the connection will be automatically closed with an `error_code` of 0 and an empty
+/// `reason`. You can also trigger this by calling `close()` on the `Connection` explicitly.
```

Comment:
```suggestion
/// `reason`. The connection can also be closed explicitly with `Connection::close()`.
```

Reply:
Hmm, I was thinking about that and think that passive voice is often best avoided. I'm also not crazy about `you`, but I do think it can make documentation easier to digest.

Reply:
(But the other parts we could change, might be better.)

Reply:
I was mostly trying to get rid of "trigger this," which struck me as a bit unclear.